tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package dagr.tasks.vc
+
+import java.nio.file.Path
+
+import dagr.commons.CommonsDef.yieldAndThen
+import dagr.commons.io.Io
+import dagr.core.config.Configuration
+import dagr.core.execsystem.{Cores, Memory}
+import dagr.core.tasksystem.{Pipe, PipeOut, ShellCommand, Task}
+import dagr.tasks.DagrDef.{FilePath, PathToBam, PathToFasta, PathToVcf}
+import dagr.tasks.DataTypes.Vcf
+import htsjdk.samtools.SamReaderFactory
+
+/**
+  * The VarDictJava directory structure is assumed to be as follows:
+  *   - Root directory: /path/to/git/VarDictJava
+  *   - Perl scripts:   /path/to/git/VarDictJava/VarDict
+  *   - Startup script: /path/to/git/VarDictJava/build/install/VarDict/bin/VarDict
+  */
+object VarDictJava extends Configuration {
+  /** Config key for the VarDictJava directory.  This should be the root project directory. */
+  val VarDictJavaDirConfigKey = "vardictjava.dir"
+
+  /** Path to the VarDictJava directory. */
+  lazy val VarDictJavaDir = configure[Path](VarDictJavaDirConfigKey)
+
+  /** Path to the VarDictJava install directory with the VarDictJava startup script. */
+  lazy val VarDictJava = VarDictJavaDir.resolve("build/install/VarDict/bin/VarDict")
+
+  /** Path to the VarDictJava bin directory with a multitude of perl scripts. */
+  lazy val VarDictJavaBinDir = VarDictJavaDir.resolve("VarDict")
```

Comment:
I found the naming confusing ... since it's nested in an object called VarDictJava how about just `BinDir`?  Also same comment here about directory structure.